src/node_util.cc

```diff
@@ -289,34 +291,66 @@ static void GuessHandleType(const FunctionCallbackInfo<Value>& args) {
   CHECK_GE(fd, 0);
 
   uv_handle_type t = uv_guess_handle(fd);
-  const char* type = nullptr;
+  uint32_t type{0};
 
   switch (t) {
     case UV_TCP:
-      type = "TCP";
+      type = 0;
```

Comment:
We should probably add a comment here explaining what these numbers are as it might not be very obvious

Reply:
I added a comment here.